docs/using-the-jdbc-driver/using-plugins/UsingTheAwsSecretsManagerPlugin.md

```diff
@@ -12,10 +12,15 @@ The following properties are required for the AWS Secrets Manager Connection Plu
 
 > **Note:** To use this plugin, you will need to set the following AWS Secrets Manager specific parameters.
 
-| Parameter                | Value  |                                                                                     Required                                                                                      | Description                                             | Example     | Default Value |
-|--------------------------|:------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------|:------------|---------------|
-| `secretsManagerSecretId` | String |                                                                                        Yes                                                                                        | Set this value to be the secret name or the secret ARN. | `secretId`  | `null`        |
-| `secretsManagerRegion`   | String | Yes unless the `secretsManagerSecretId` is a Secret ARN. A Secret ARN has the following format: `arn:aws:secretsmanager:<Region>:<AccountId>:secret:SecretName-6RandomCharacters` | Set this value to be the region your secret is in.      | `us-east-2` | `us-east-1`   |
+| Parameter                | Value  |                         Required                         | Description                                             | Example     | Default Value |
+|--------------------------|:------:|:--------------------------------------------------------:|:--------------------------------------------------------|:------------|---------------|
+| `secretsManagerSecretId` | String |                           Yes                            | Set this value to be the secret name or the secret ARN. | `secretId`  | `null`        |
+| `secretsManagerRegion`   | String | Yes unless the `secretsManagerSecretId` is a Secret ARN. | Set this value to be the region your secret is in.      | `us-east-2` | `us-east-1`   |
+
+*NOTE* A Secret ARN has the following format: `arn:aws:secretsmanager:<Region>:<AccountId>:secret:SecretName-6RandomCharacters`
+
+## Secret Data
+The plugin assumes that the secret contains the following properties `username` and `password`
```

Comment:
```suggestion
The plugin assumes that the secret contains the following properties `user` and `password`
```

Reply:
https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/0bf0c68aef99e7b84368870848ca1c0c5befb5a1/wrapper/src/main/java/software/amazon/jdbc/plugin/AwsSecretsManagerConnectionPlugin.java#L283 suggests otherwise or am I mis-understanding?